### DISCUSSION TITLE
TypeScript - Speed

### DISCUSSION THREAD

**Discussion by PPetterssonSO** [SuperOffice/typescript-for-superoffice#24] [General]
Hi,
This post might be better in the Community, but here goes.
Saving / running TypeScripts are very very slow, saving a 1 line crmscript takes 0.x seconds, saving a one line TypeScript takes up to 5 seconds.

Is it anything we can do to fix this?

**Comment by ejfasting** [comment]
Hello @Nawksen, 

The reason behind this is that the typescript gets compiled into javascript when you save it. 
This compile-time has been noted as one of the things we need to improve, and we did bump the resources yesterday to improve it a little. 

So this is how it is for now, but its on the roadmap to figure out how to best handle it. 

//Eivind
